java/mapping/supported.js

```diff
@@ -16,7 +16,9 @@
     'variationGroupId',
     'hooks',
     'timeout',
-    'pageId']
+    'pageId',
+    'waitBeforeCapture'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9461648)
```suggestion
    "waitBeforeCapture"
```